pagefaultgames/pokerogue
# Berry type move

## Why am I doing these changes?



Including an entire class just to get enum definitions can create a nasty circular dependency chain which can cause a crash. 

## What did change?


This change extracts the BerryType enum into its own file so that other files that import it never run into an issue. berry-type.ts has no imports and therefore cannot cause circular dependency errors.

## How to test the changes?




This is a code change which does not affect functionality. `npm run test` succeeding and the game launching should suffice as the refactor was done by the IDE exclusively.

## Checklist
